dotnet/src/SemanticKernel/AI/ChatMessageGeneration/IChatCompletion.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.AI.ChatMessageGeneration;
+
+public interface IChatCompletion
```

Comment:
Inconsistent naming IChatCompletion vs ChatMessageGeneration...
Are we billing this as a "completion" service or a "generation" service?